src/components/NftCard/NftCard.tsx

```diff
@@ -28,7 +34,7 @@ export const NftCard = ({ nft, marketplace }: NftCardProps) => {
           className="w-full aspect-square object-cover"
           src={nft.image as string}
         />
-        {nft.offers.length > 0 && (
+        {nft.offers && nft.offers.length > 0 && (
```

Comment:
Why not just the length check? offers isn't empty if none are found?

Reply:
offers can be 'undefined'